frontend/src/components/workflow-list.ts

```diff
@@ -553,7 +549,7 @@ export class WorkflowList extends LitElement {
 
   render() {
     return html` <div class="listHeader row">
-        <div class="col">${msg("Name & Last Updated")}</div>
+        <div class="col">${msg("Workflow Name & Last Updated")}</div>
```

Comment:
I think the intent for this change previously was to just use 'Name' to avoid using 'Workflow' without 'Crawl Workflow', and seemed shorter..

Reply:
Should we rename this page to "Crawl Workflows" then? There's no indication that this list is of crawl workflows, besides assumed based on "New Crawl Workflow"

Reply:
+1 for renaming the page and section title to "Crawl Workflows" so that the name of the page is in-line with all of our other main pages that are named based on the name of the objects they contain.